ouster_client/src/lidar_scan.cpp

```diff
@@ -9,6 +9,7 @@
 #include <cmath>
 #include <cstddef>
 #include <cstring>
+#include <iostream>
```

Comment:
I think `sstream` should have been sufficient.

Reply:
good point, we'll get it on the next one